image_geometry/include/image_geometry/pinhole_camera_model.h

```diff
@@ -270,7 +270,7 @@ class PinholeCameraModel
 
   // Use PIMPL here so we can change internals in patch updates if needed
   struct Cache;
-  boost::shared_ptr<Cache> cache_; // Holds cached data for internal use
+  std::shared_ptr<Cache> cache_; // Holds cached data for internal use
```

Comment:
Yeah, C++11 and no boost is more important for ROS2.

You are right that this is an ABI break; will that be a problem for ROS1?